docs/contrib/PlatformBringUp.md

```diff
@@ -0,0 +1,531 @@
+# User Guide to Integrate a New Platform into the OpenROAD Flow
+
+# Table of Contents <a name="TOC"></a>
+
+- [Table of Contents](#TOC)
+- [Overview](#Overview)
+- [Prerequisites](#Prerequisites)
+- [Adding a new Platform to OpenROAD](#APOpenROAD)
+	- [Setup](#Setup)
+		- [Makefile](#Makefile)
+		- [Platform Directory](#PlatformDirectory)
+		- [Design Directory](#DesignDirectory)
+	- [Platform Configuration](#PlatformConfiguration)
+	- [Design Configuration](#DesignConfiguration)
+		- [config.mk](#configmk)
+		- [constraint.sdc](#constraintsdc)
+		- [Liberty, LEF, and GDS Files](#LibertyLEFGDSFiles)
+	- [Behavioral Models](#BehavioralModels)
+		- [Gated Clock](#GatedClock)
+		- [Latch](#Latch)
+	- [FastRoute Configuration](#FastRouteConfiguration)
+	- [Metal Tracks Configuration](#MetalTracksConfiguration)
+	- [PDN Configuration](#PDNConfiguration)
+	- [Tapcell Configuration](#TapcellConfiguration)
+	- [setRC Configuration](#setRCConfiguration)
+	- [KLayout](#KLayout)
+		- [KLayout properties file](#KLayoutpropertiesfile)
+		- [KLayout tech file](#KLayouttechfile)
+- [Validating the New Platform](#ValidatingPlatform)
+- [Authors/Contributors](#AuthorsContributors)
+
+
+# Overview <a name="Overview"></a>
+
+This document is a guide for foundry and third party IP providers to easily integrate and test a new technology into the OpenROAD RTL to GDS flow. OpenROAD allows you to integrate any public PDK (Process Design Kit) for any feature size and implement a fully open-sourced RTL-GDSII flow (synthesizable Verilog to merged GDSII). The OpenROAD flow has been validated for feature sizes up to 7nm and used to design and tapeout over 100 ASIC and SoCs to date.
```

Comment:
In: integrate and test a new technology **into** the OpenROAD RTL to GDS flow.

into -> in